addons/mail/static/src/core/web/message_actions_patch.js

```diff
@@ -1,22 +1,113 @@
+import { getNonEditableMentions, parseEmail } from "@mail/utils/common/format";
 import { messageActionsRegistry } from "@mail/core/common/message_actions";
 import { _t } from "@web/core/l10n/translation";
+import { renderToMarkup } from "@web/core/utils/render";
+import { rpc } from "@web/core/network/rpc";
+
+function openFullComposer(title, context, actionService, thread, message) {
+    const action = {
+        name: title,
+        type: "ir.actions.act_window",
+        res_model: "mail.compose.message",
+        view_mode: "form",
+        views: [[false, "form"]],
+        target: "new",
+        context: {
+            ...context,
+            default_model: thread.model,
+            default_res_ids: [thread.id],
+            default_subject: message.subject || message.default_subject,
+            default_subtype_xmlid: "mail.mt_comment",
+        },
+    };
+    actionService.doAction(action, {
+        onClose: () => thread.fetchNewMessages(),
+    });
+}
```

Comment:
Could pass `component` instead of `actionService, thread,  message`.
The less arguments the better, notably for params that are coupled to a bigger concern like `component`.